icefoganalytics/travel-authorization
# Cleanup: Merge `data` and `db` Folders

# Context

Currently there are two database related folders `api/src/db` and `api/src/data`. I'm finally making the time to merge these folders and make everything database related a little easier to find.

# Implementation

1. This required updating api/src/config Knex migration and seeds paths. 
2. Required updating seed paths in package.json build script. 
3. I can't seem to find any references to the data examples, but I've kept them in the simplified db/data directory.

# Testing Instructions

1. Run the test suite via `dev test` (or `dev test_api`)
2. Boot the app via `dev up`
3. Log in to the app at http://localhost:8080
4. Check that you can run migrations at http://localhost:3000/migrate/up and http://localhost:3000/migrate/down
5. Check that you can run seeds at http://localhost:3000/migrate/seed
6. Build/boot the app in production mode via `dev up --build`
7. Check that you can log in at http://localhost:3000
8. Check that you can run migrations at http://localhost:3000/migrate/up and http://localhost:3000/migrate/down
9. Check that you can run seeds at http://localhost:3000/migrate/seed
